vortex-datafusion/src/lib.rs

```diff
@@ -50,12 +50,12 @@ const SUPPORTED_BINARY_OPS: &[Operator] = &[
 /// Optional configurations to pass when loading a [VortexMemTable].
 #[derive(Default, Debug, Clone)]
 pub struct VortexMemTableOptions {
-    pub disable_pushdown: bool,
+    pub enable_pushdown: bool,
```

Comment:
If we do this we need to impl default by hand

Reply:
nice catch, I'll add that to the file provider PR